plugins/exporter.koplugin/main.lua

```diff
@@ -299,14 +299,21 @@ For more information, please visit https://github.com/koreader/koreader/wiki/Hig
                     return not self:isDocless() and self:readyToExport() and not self.txt_export
                 end,
                 callback = function()
-                    UIManager:scheduleIn(0.5, function()
-                        self:exportCurrentNotes(self.view)
-                    end)
-
-                    UIManager:show(InfoMessage:new{
-                        text = _("Exporting may take several seconds…"),
-                        timeout = 1,
-                    })
+                    local export_callback = function()
+                        UIManager:scheduleIn(0.5, function()
```

Comment:
Hm, is this some remaining old paradigm? tickAfterNext should probably do the trick (or just nextTick).

Reply:
Looking around, I think so. I'm seeing examples in the code of `scheduleIn(0.5, ...) from 2+ years ago, and nextTick and tickAfterNext came around 10 months ago.

Reply:
nextTick seems to work fine. I'm not sure if there are any considerations I'm missing that would lead me to use tickAfterNext instead.

Reply:
In any case, this plugin goes back to 2014 and it hasn't seen much if any love by most of us. ;-)